js/pages/RetirementReportCardDemo.jsx

```diff
@@ -6,20 +6,34 @@ import SolutionSpecificDemo from '../components/SolutionSpecificDemo';
 
 import Page from '../components/Page';
 import WidgetBox from '../components/WidgetBox';
+import SimpleSpacer from '../components/SimpleSpacer';
 
-const RetirementReportCardWidget = () => (
-  <WidgetBox width="small" height="auto">
-    <NextCapitalComponent
-      getEmbed={
-        (client) => new client.RetirementReportCard({
-          onButtonClick: (targetState) => window.alert(
-            `button clicked! expected to perform SSO for target state: ${targetState}`
-          )
-        })
-      }
-      loadingContent="loading..."
-    />
-  </WidgetBox>
+const RetirementReportCardWidget = ({ context = null, title }) => (
+  <div>
+    <h2>{ title }</h2>
+    <WidgetBox width="small" height="auto">
+      <NextCapitalComponent
+        getEmbed={
+          (client) => new client.RetirementReportCard({
+            onButtonClick: (targetState) => window.alert(
+              `button clicked! expected to perform SSO for target state: ${targetState}`
+            ),
+            context
```

Comment:
```suggestion
          (client) => new client.RetirementReportCard({
            context,
            onButtonClick: (targetState) => window.alert(
              `button clicked! expected to perform SSO for target state: ${targetState}`
            )
```
slightly more readable